README.md

```diff
@@ -106,3 +106,21 @@ Beyond the officially supported [flow environments](https://docs.signac.io/proje
 
 * Fry
 * Kestrel
+
+## Included shell scripts
+<details>
+     <summary>clean.sh (Click to expand)</summary>
+clean.sh is a shell script file that will remove the workspace folder, including the files inside of it, along with any files matching status.txt, signac*, *.log, *.gsd, or *.out within the present planckton-flow clone you are using. To run this shell script:
+
+    bash clean.sh 
+
+in your command line. **THIS WILL DELETE ANY JOBS YOU HAVE RUN SO FAR! MAKE SURE YOU ARE IN THE CLONE OF PLANCKTON-FLOW THAT YOU WANT THE WORKSPACE REMOVED FROM!**
+</details>
```

Comment:
```suggestion
<details>
     <summary>clean.sh</summary>
`clean.sh` is a shell script file that will remove the workspace folder, including the files inside of it, along with any files matching `status.txt`, `signac*`, `*.log`, `*.gsd`, or `*.out` within the present planckton-flow clone you are using. To run, type the following in your terminal:

    bash clean.sh 

**THIS WILL DELETE ANY JOBS YOU HAVE RUN SO FAR! MAKE SURE YOU ARE IN THE CLONE OF PLANCKTON-FLOW THAT YOU WANT THE WORKSPACE REMOVED FROM!**
</details>
```